src/index.tsx

```diff
@@ -2,12 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
+import log from 'loglevel';
 
 import './styles.scss';
 import App from './components/App';
 import configureStore from './configureStore';
 import { actions as apiActions } from './reducers/api';
 
+if (process.env.NODE_ENV === 'production') {
+  log.setLevel(log.levels.INFO);
```

Comment:
This should probably be `log.setLevel(..., false)` to disable persistence. Otherwise, the docs say that the lib will try to store the level in cookies or localStorage. I don't think we need that.

Reply:
oh, the level gets persisted! Is that a problem though?